### PR TITLE
Transcript in output

### PR DESCRIPTION
**Pull request by jason-raitz**
see AP-271. 

If the string "transcript" shows up in an 856 42$y value, then we use that value as the link text and the corresponding $u for the url value.  If there are multiple transcripts, it will place a link for each one.  Note: if there actually are more than one transcript for a given record, it would be a good practice to disambiguate them in their $y link descriptions.

### CODE REVIEW

File: app/views/player/_record.html.erb [hunk 1]
```diff
@@ -86,6 +85,31 @@
...
+      <% if transcripts.any? %>
+        <tr>
+          <th><p>Transcripts</p></th>
+          <td>
+            <% transcripts.each do |transcript| %>
+              <%= link_to(transcript['y'], transcript['u']) %>
+            <% end %>
+          </td>
+        </tr>
+      <% end %>
```
Comment (anarchivist) [inline app/views/player/_record.html.erb:112]: i'm a little concerned that this adds a lot of logic to the view, especially since  the preceeding section is already doing something with the metadata.

can this logic be moved somewhere into `BerkeleyLibrary::AV::Metadata`? it seems like we can probably add a `#transcripts` method.

File: docker-compose.yml [hunk 2]
```diff
@@ -15,3 +13,5 @@ services:
       # Note that this mounts the *entire* repo directory (including
       # files ignored in .dockerignore when building the image)
       - ./:/opt/app
+    tty: true
+    stdin_open: true
```
Comment (anarchivist) [inline docker-compose.yml:17]: are these changes necessary?

**Review by anarchivist — CHANGES_REQUESTED** [review]
the logic in the view feels a bit heavy - we may want to extend our code in the av-core gem instead to make it easier for us to pull out the data we need than keeping the logic in the view.

File: docker-compose.yml (hunk 2 above)
Comment (jason-raitz) [reply docker-compose.yml:17]: these may not be necessary, but they do help with debugging.

**Review by anarchivist — APPROVED** [review]
